study.md

```diff
@@ -54,5 +54,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+I'm still not fully sure on what an enumerable is, but in terms of differences it looks like a stream can't use the same types of functions because of the lack of limitations since it's codata.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.